### DISCUSSION TITLE
Will turbopack be usable as a Rust library?

### DISCUSSION THREAD

**Discussion by brundonsmith** [vercel/turborepo#3732] [Help]
I've learned that `swc_bundler` has been deprecated in favor of turbopack, however, the former is available as a Rust library on crates.io, has docs on docs.rs, etc, while I couldn't find turbopack on crates.io or any instructions on how to use it as a library

I appreciate that turbpoack is a pretty new project, but I was wondering what the future plans/roadmap are, if any, for making turbopack available as a Rust library that others can build tools on top of?

Thanks!

**Comment by juancampa** [comment]
I'm also interested in this question
